### PR TITLE
Brad/image alignment

### PR DESCRIPTION
**Pull request by bbraithwaite**
Internal change.

//cc @AdamSC1-ddg

### CODE REVIEW

File: views/email/a/v1.tx [hunk 1]
```diff
@@ -1,5 +1,5 @@
                                      <p style="margin-bottom: 1em">Recently we found that <b>66%</b> of people who use Private Browsing (aka "Incognito Mode") over-estimate the protection it provides:</p>
-<p style="margin-bottom: 1em; padding-top: 10px; padding-left:10px"><img src="https://duckduckgo.com/assets/email/v1.png?atb=<: $subscriber.flow :>" style="width:100%; padding-left: 5px; padding-bottom: 10px; color: #333333; font-size: 14px; font-family: sans-serif; font-style:italic; background: #ffffff" alt="Disabled Image: Chart of misconceptions about Private Browsing Mode"/>
+<p style="margin-bottom: 1em; padding-top: 10px; padding-left:0;"><img src="https://duckduckgo.com/assets/email/v1.png?atb=<: $subscriber.flow :>" style="width:100%; padding-left: 5px; padding-bottom: 10px; color: #333333; font-size: 14px; font-family: sans-serif; font-style:italic; background: #ffffff" alt="Disabled Image: Chart of misconceptions about Private Browsing Mode"/>
```
Comment (bbraithwaite) [inline views/email/a/v1.tx]: removes the left-padding to look as:

screen shot 2017-04-19 at 16 27 13

File: views/email/a/v2.tx [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 
                         <p style="margin-bottom: 1em">Advertising networks have hidden tracks on most internet sites. Google alone has trackers on 75% of the top million sites! Unless you prevent it, you're being tracked on all these sites. Then, your profile is made available for advertisers to bid on, allowing them to target you with their ads wherever you go:</p>
-<p style="margin-bottom: 1em; padding-top: 5px; padding-bottom: 15px; padding-left:10px"><img src="https://duckduckgo.com/assets/email/v2.png?atb=<: $subscriber.flow :>" style="width:100%; padding-left: 5px; padding-bottom: 15px; color: #333333; font-size: 14px; font-family: sans-serif; font-style:italic; background: #ffffff" alt="Disabled Image: Headline from Wall Street Journal Article"/></p>
+<img src="https://duckduckgo.com/assets/email/v2.png?atb=<: $subscriber.flow :>" style="width:100%; padding-left: 5px; padding-bottom: 15px; color: #333333; font-size: 14px; font-family: sans-serif; font-style:italic; background: #ffffff" alt="Disabled Image: Headline from Wall Street Journal Article"/>
```
Comment (bbraithwaite) [inline views/email/a/v2.tx]: **Before**
screen shot 2017-04-19 at 16 28 48

**After**
screen shot 2017-04-19 at 16 29 08

File: views/email/a/v3.tx [hunk 3]
```diff
@@ -3,6 +3,5 @@
                         <p style="margin-bottom: 1em"><a href="https://myactivity.google.com/myactivity">https://myactivity.google.com/myactivity</a></br></br>
         <p style="margin-bottom: 1em">It's just plain creepy, and in 2012, Google made a change to their privacy policy that enabled them to combine all this data together into one master profile on you, just to better target ads at you across the web.</br></br>
 
-Privacy Tip — Delete your Google activity on the "My Activity" page:</br><ul>
+Privacy Tip — Delete your Google activity on the "My Activity" page:</br>
```
Comment (bbraithwaite) [inline views/email/a/v3.tx]: **Before**

screen shot 2017-04-19 at 16 20 11


**After**

screen shot 2017-04-19 at 16 21 01

**Comment by jbarrett** [comment]
Can stage this and launch a test tun later @bbraithwaite @AdamSC1-ddg

**Pull request merged**